src/wish.cpp

```diff
@@ -200,13 +210,21 @@ class wish_mutate_callback: public uilist_callback
 
             lastlen = line2 + 1;
 
-            mvwprintz( menu->window, point( startx, menu->w_height - 3 ), c_green, msg );
+            mvwprintz( menu->window, point( startx, menu->w_height - 4 ), c_green, msg );
             msg.clear();
             input_context ctxt( menu->input_category, keyboard_mode::keycode );
-            mvwprintw( menu->window, point( startx, menu->w_height - 2 ),
+            mvwprintw( menu->window, point( startx, menu->w_height - 3 ),
                        _( "[%s] find, [%s] quit, [t] toggle base trait" ),
                        ctxt.get_desc( "FILTER" ), ctxt.get_desc( "QUIT" ) );
 
+            if( only_active ) {
+                mvwprintz( menu->window, point( startx, menu->w_height - 2 ), c_green,
+                           _( "[a] show active traits (active)" ) );
+            } else {
+                mvwprintz( menu->window, point( startx, menu->w_height - 2 ), c_white,
+                           _( "[a] show active traits         " ) );
```

Comment:
I guess you don't really need spaces here.
```suggestion
                           _( "[a] show active traits" ) );
```

Reply:
They are needed to overwrite the `(active)` with blank space.

Reply:
I removed spaces, tested locally and there's no problem.

Reply:
Same. I don't know what test I did in which the space was necessary. Weird.